include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -563,164 +590,6 @@ namespace internal
       }
     };
 
-    template <typename Number>
-    struct EvaluatorTypeTraits<1, 1, Number>
-    {
-      using ScalarNumber =
```

Comment:
Note that I removed this partial specialization, because with `spacedim` being another parameter, I would have needed even more cases. Instead, I now added an `std::enable_if_t` to the specialization with `n_components == dim` that further checks `dim > 1` to avoid ambiguous specializations for `n_components = 1` and `n_components = dim`. I think this gives us cleaner code.